services/API-service/src/scripts/mock.service.ts

```diff
@@ -481,20 +497,27 @@ export class MockService {
     ].includes(disasterType);
   }
 
-  private shouldMockExposedAssets(disasterType: DisasterType): boolean {
-    return disasterType === DisasterType.FlashFloods;
-  }
-
-  private shouldMockDynamicPointData(
+  private shouldMockExposedAssets(
     disasterType: DisasterType,
-    scenarioName: string,
+    countryCodeISO3: string,
   ): boolean {
     return (
-      disasterType === DisasterType.FlashFloods &&
-      scenarioName !== FlashFloodsScenario.NoTrigger
+      disasterType === DisasterType.FlashFloods && countryCodeISO3 === 'MWI' // TODO: make this conditional based on layers (e.g. roads) being configured for the country/disaster-type
     );
   }
 
+  private async shouldMockRiverGaugeData(
+    layers: LayerMetadataEntity[],
+    scenarioName: string,
+  ): Promise<boolean> {
+    console.log('layers: ', layers);
+    const hasRiverGaugeLayer = layers
+      .map((layer) => layer.name)
+      .includes(PointDataCategory.gauges);
+    console.log('hasRiverGaugeLayer: ', hasRiverGaugeLayer);
```

Comment:
remove `console.log` or at least format it such that it's useful in prod

Reply:
removed, this was unintended